### DISCUSSION TITLE
Not finding this folder chaincode-docker-devmode

### DISCUSSION THREAD

**Discussion by aruns05** [hyperledger/fabric-samples#824] [Q&A]
Hello All

I am going through the below section. it says git clone fabric-samples will also add a folder chaincode-docker-devmode
I am not able to see it, can anyone advise

https://github.com/hyperledger/fabric-contract-api-go/blob/v1.1.1/tutorials/getting-started.md

**Comment by deefstes** [comment]
I'm having this same issue going through the [Getting started guide for v1.2.1](https://github.com/hyperledger/fabric-contract-api-go/blob/v1.2.1/tutorials/getting-started.md). Were you ever able to work out where to find this folder?
